src/components/AddComment.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useContext } from "react";
+import { addCommentByArticle } from "../api";
+import { UserContext } from "../contexts/UserContext";
+
+const AddComment = ({ article_id }) => {
+  const { loggedInUser } = useContext(UserContext);
+
+  const [newComment, setNewComment] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    addCommentByArticle(article_id, loggedInUser.username, newComment);
+    setNewComment("");
+    let commentInput = document.getElementById("new-comment-textbox");
+    commentInput.value = "";
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <textarea
+        name="textbox"
+        id="new-comment-textbox"
+        cols="30"
+        rows="3"
+        placeholder="Add comment..."
+        onChange={(event) => {
+          setNewComment(event.target.value);
+        }}
+      ></textarea>
```

Comment:
if you want to create a controlled component don't forget to reflect the value that's held in state back into the input!

`value={newComment}`